browser/brave_vpn/sources.gni

```diff
@@ -15,7 +15,14 @@ if (enable_brave_vpn) {
     "//brave/browser/brave_vpn/vpn_utils.cc",
     "//brave/browser/brave_vpn/vpn_utils.h",
   ]
-
+  if (is_win) {
+    brave_browser_brave_vpn_sources += [
+      "//brave/browser/brave_vpn/dns/brave_vpn_dns_observer_factory.cc",
+      "//brave/browser/brave_vpn/dns/brave_vpn_dns_observer_factory.h",
+      "//brave/browser/brave_vpn/dns/brave_vpn_dns_observer_service.cc",
+      "//brave/browser/brave_vpn/dns/brave_vpn_dns_observer_service.h",
+    ]
+  }
```

Comment:
Do we need to add GN deps to `brave_browser_brave_vpn_deps` for files included by brave_vpn_dns_observer_factory / brave_vpn_dns_observer_service? 

I ran `npm run gn_check`, so they're likely already included in `src/brave/browser/sources.gni`; but I spot checked and didn't see `//components/country_codes:country_codes` in there

Reply:
done